src/pages/HomePage.integration.test.tsx

```diff
@@ -0,0 +1,76 @@
+import { render, screen, waitFor } from '@testing-library/react';
```

Comment:
you don't need this file, as you're already testing API call in your homePage.test.tsx

Reply:
You can move some missing parts from this file into the HomePage.test.tsx file